### PR TITLE
Exif handling

### LINKED ISSUE #13: Strip EXIF metadata from uploaded images
Stuff like exact date/time, location and a bunch of other things. We don't need it for anything, so we shouldn't store it.

### PR DESCRIPTION
**Pull request by duijf**
Fixes #13 

We're not going to support any EXIF metadata for now, but we will upload/handle the `Orientation` tag correctly before stripping it.

### CODE REVIEW

File: pxl/compress.py [hunk 1]
```diff
@@ -47,3 +49,35 @@ def compress_image(local_filename: Path) -> Dict[state.Size, Path]:
...
     return image_paths
+
+
+def orient_exif(image: Any) -> Any:
+    """
+    Rotate the image according to EXIF metadata.
+    """
+    exif_data = image._getexif()
+
+    orientation_tag = 274
```
Comment (tomwassenberg) [inline pxl/compress.py:65]: Can you add a comment (maybe with link to spec) to explain the magic number here?

**Review by tomwassenberg — COMMENTED** [review]
I would _very much_ like one more comment, rest LGTM :+1:

File: pxl/compress.py (hunk 1 above)
Comment (duijf) [reply pxl/compress.py:65]: The spec link isn't too easy (it's a PDF): https://www.exif.org/Exif2-2.PDF search for "Orientation"

Comment (duijf) [reply pxl/compress.py:65]: But I documented this

**Pull request merged**